Slack.php

```diff
@@ -120,6 +126,20 @@ function bugnote_add_form($event, $bug_id) {
         echo '</td></tr>';
     }
 
+    function get_color($status){
+        // split the status_enum string which is a string like 10:new,20:open, ...
+        $status_array = explode (",", config_get('status_enum_string'));
+        foreach ($status_array as $elem){
+            $elem = explode (":",$elem);
+            // if $status  match the  status number from my config I search the color that match the name of the status 
+            if ($elem[0] == $status){
```

Comment:
needs an additional condition to ensure this entry exists in the `status_colors` settings.